model/pull_request.go

```diff
@@ -27,5 +28,5 @@ type PullRequest struct {
 	BuildLink           string
 	URL                 string
 	CreatedAt           time.Time
-	MaintainerCanModify bool
+	MaintainerCanModify sql.NullBool
```

Comment:
Hmm.. this is okay for now, but I'll prefer a separate mapping which maps SQL objects to model objects. Right now, this leaks the abstraction that `PullRequest` is tied to a SQL query which does not make it very flexible.

We can scan a separate object having `sql.NullBool` and then map it to PullRequest which can have a normal `bool` pointer field.

Reply:
Agree